src/main/java/org/takes/rq/RqWithoutHeader.java

```diff
@@ -49,17 +50,18 @@ public RqWithoutHeader(final Request req, final CharSequence name) {
         super(
             // @checkstyle AnonInnerLengthCheck (50 lines)
             new RequestOf(
-                () -> {
-                    final Text prefix = new FormattedText(
-                        "%s:",
-                        new Lowered(name.toString()).asString()
-                    );
-                    return new Filtered<>(
-                        header -> !new Lowered(header).asString()
-                            .startsWith(prefix.asString()),
-                        req.head()
-                    );
-                },
+                () -> new Filtered<>(
```

Comment:
@fanifieiev Wouldn't be possible to work only with `Text` here, without converting them to strings / evaluating it by  `value` method , to use advantage of the lazy evaluating?

Reply:
@paulodamaso For now `Filtered`  accepts `Func<X, Boolean>`, so it is not possible to do what you ask for.
Instead the  `Filtered`  has to accept `Func<X, Scalar<Boolean>>`, then it will work.

I can create an issue in Cactoos project.